cg/apps/gisaid/gisaid.py

```diff
@@ -0,0 +1,126 @@
+"""Interactions with the gisaid cli upload"""
+
+import logging
+from pathlib import Path
+from typing import List
+
+from cg.store import models
+from cg.meta.workflow.fastq import FastqHandler
+from ...utils import Process
```

Comment:
I have the solution to this, can show you later if you are interested

Reply:
Yes please